features/storage/csi_snapshot.feature

```diff
@@ -53,5 +53,71 @@ Feature: Volume snapshot test
     And the output should contain "snapshot test"
     
     Examples:
-      | csi-sc  |
-      | gp2-csi | # @case_id OCP-27727
+      | csi-sc       |
+      | gp2-csi      | # @case_id OCP-27727
+      | standard-csi | # @case_id OCP-37568
+
+
+  # @author wduan@redhat.com
+  @admin
+  Scenario Outline: Volume snapshot create and restore test with block
+    Given I have a project
+    Given I obtain test data file "storage/misc/pvc.json"
+    When I create a dynamic pvc from "pvc.json" replacing paths:
+      | ["metadata"]["name"]         | mypvc-ori |
+      | ["spec"]["storageClassName"] | <csi-sc>  |
+      | ["spec"]["volumeMode"]       | Block     |
+    Then the step should succeed
+    Given I obtain test data file "storage/misc/pod-with-block-volume.yaml"
+    When I run oc create over "pod-with-block-volume.yaml" replacing paths:
+      | ["metadata"]["name"]                                         | mypod-ori   |
+      | ["spec"]["volumes"][0]["persistentVolumeClaim"]["claimName"] | mypvc-ori   |
+      | ["spec"]["containers"][0]["volumeDevices"][0]["devicePath"]  | /dev/dblock |
+    Then the step should succeed
+    And the pod named "mypod-ori" becomes ready
+    When I execute on the pod:
+      | /bin/dd | if=/dev/zero | of=/dev/dblock | bs=1M | count=1 |
```

Comment:
Need a bit of explanation about what this step do.

Reply:
Added comments to explain the steps.